homeassistant/components/uptime/sensor.py

```diff
@@ -50,4 +50,4 @@ def __init__(self, name: str) -> None:
         self._attr_name: str = name
         self._attr_device_class: str = DEVICE_CLASS_TIMESTAMP
         self._attr_should_poll: bool = False
-        self._attr_native_value: str = dt_util.now().isoformat()
+        self._attr_native_value = dt_util.now()
```

Comment:
Sidd note: Looks like this is incorrectly returning local time instead of UTC time.